release-notes/opendistro-for-elasticsearch.job-scheduler.release-notes-1.10.0.0.md

```diff
@@ -0,0 +1,4 @@
+## 2020-08-18, version 1.10.0.0
+
+### New Features
+* Adds support for Elasticsearch 7.9.0 - [PR #67](https://github.com/opendistro-for-elasticsearch/job-scheduler/pull/67)
```

Comment:
Do the changes in JobSchedulerPlugin break existing plugins and require them to change their own code at all? If so we should add it as breaking changes in the release notes.

Reply:
Thank you for pointing that out! I will think it.

Reply:
Seems to be fine, tested with ISM